stdlib/pagerduty/pagerduty.flux

```diff
@@ -330,6 +330,9 @@ endpoint = (url=defaultURL) =>
                                 customDetails: record.get(r: obj, key: "customDetails", default: record.any),
                             )
 
-                        return {r with _sent: string(v: 2 == status / 100), _status: string(v: status)}
+                        return {r with _sent: string(v: 2 == response.statusCode / 100),
+                            _status: string(v: response.statusCode),
+                            _body: string(v: response.body),
+                        }
```

Comment:
This does change the pattern of all the existing `endpoint` functions. I'm not against it, but it likely needs to be considered a breaking change since it will affect downstream use cases.

Reply:
Technically it's not a breaking change since it's additive (only adding the response body to `_body`). Pagerduty is already unique to other endpoints in that it includes the `_sent` field in addition to the apparent default of `_status`.

Reply:
Good point.